assets/components/LogViewer/LogViewer.vue

```diff
@@ -15,7 +16,7 @@
       >
         <ic:sharp-find-in-page />
       </a>
-      <component :is="item.getComponent()" :log-entry="item" :visible-keys="visibleKeys" />
+      <component :is="item.getComponent()" :log-entry="item" :visible-keys="visibleKeys" :is-hovering="true" />
```

Comment:
I don't understand what this is doing. It is hardcoded to `true`.

Reply:
It was a testing thing. Cleaned up!